README.md

```
@@ -1,210 +1,216 @@
-# Install specific applications
-./categories/security/wazuh.sh          # Install Wazuh SIEM
-./categories/security/usb-blocker.sh    # USB Port Security Tool
-./categories/development/nvm.sh         # Node Version Manager
-./categories/devops/docker.sh          # Latest Docker Engine
+./multi-select-installer.sh
 ```
 
-## Quick Install Stacks
+### Step 2: Main Menu Navigation
+You'll see the main menu:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)